net-libs/qxmpp/qxmpp-1.0.0.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils
+
+DESCRIPTION="A cross-platform C++ XMPP client library based on the Qt framework"
+HOMEPAGE="https://github.com/qxmpp-project/qxmpp/"
+SRC_URI="https://github.com/${PN}-project/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug doc opus +speex test theora vpx"
+
+RDEPEND="
+	dev-qt/qtcore
+	dev-qt/qtnetwork[ssl]
+	dev-qt/qtxml
+	opus? ( media-libs/opus )
+	speex? ( media-libs/speex )
+	theora? ( media-libs/libtheora )
+	vpx? ( media-libs/libvpx:= )
+"
+DEPEND="${RDEPEND}
+	test? ( dev-qt/qttest:5 )
```

Comment:
This should also be a `BDEPEND` I think

Reply:
That's a library and not just a tool (just as qtcore or qtgui expose), so should it go to `BDEPEND` still or remain in `DEPEND`?

Reply:
I'm arguing for `BDEPEND` because it's a library that only gets linked to an artifact that's discarded, i.e. to the test binary. You don't need it to link/use/run the library, but only to link the tests that are run at build time. Deferring to @juippis @mgorny

Reply:
qttest is always in DEPEND for header use.